engine/schema/src/main/resources/META-INF/db/schema-41100to41200.sql

```diff
@@ -0,0 +1,70 @@
+-- IOPS per GB disk offering
+DROP PROCEDURE IF EXISTS `cloud`.`ADD_COLUMN_TO_TABLE_IDEMPOTENT`;
+
+DELIMITER //
+CREATE PROCEDURE `cloud`.`ADD_COLUMN_TO_TABLE_IDEMPOTENT`(
```

Comment:
Isn't this being introduced by some other PR?

Reply:
No. This is only for this PR

Reply:
Hmm.. Then, I think it is better to coordinate with PR #2449.